### PR TITLE
config updates for 24.0

### PR DESCRIPTION
**Pull request by bgruening**
xref: https://github.com/usegalaxy-eu/issues/issues/500

### CODE REVIEW

File: files/galaxy/config/datatypes_conf.xml [hunk 1]
```diff
@@ -231,22 +242,21 @@
...
     <datatype extension="gatk_recal" type="galaxy.datatypes.tabular:Tabular" subclass="true" display_in_upload="true"/>
+    <datatype extension="cnn" type="galaxy.datatypes.tabular:Tabular" subclass="true" display_in_upload="true"/>
+    <datatype extension="cns" type="galaxy.datatypes.tabular:Tabular" subclass="true" display_in_upload="true"/>
+    <datatype extension="cnr" type="galaxy.datatypes.tabular:Tabular" subclass="true" display_in_upload="true"/>
     <datatype extension="hhr" type="galaxy.datatypes.data:Text" subclass="true" display_in_upload="true"/>
-    <datatype extension="jpg" type="galaxy.datatypes.images:Jpg" mimetype="image/jpeg" display_in_upload="true"/>
-    <datatype extension="tiff" type="galaxy.datatypes.images:Tiff" mimetype="image/tiff" display_in_upload="true"/>
+    <datatype extension="jpg" type="galaxy.datatypes.images:Jpg" mimetype="image/jpeg"/>
+    <datatype extension="tiff" type="galaxy.datatypes.images:Tiff" mimetype="image/tiff" display_in_upload="true">
+      <display file="image/avivator.xml"/>
```
Comment (mira-miracoli) [inline files/galaxy/config/datatypes_conf.xml:251]: Can avivator use both, ome.tiff and tiff?

File: files/galaxy/config/user_preferences_extra_conf.yml [hunk 2]
```diff
@@ -1,246 +1,274 @@
...
-
-  gitlabfs:
-    description: Your DataPLANT ARC Gitlab access details https://git.nfdi4plants.org
-    inputs:
-      - name: read_access_token
-        label: API Read Access Token
-        type: password
-        required: False
+    webhook_overlay_search:
+        description: Configuration option for your search results
```
Comment (mira-miracoli) [inline files/galaxy/config/user_preferences_extra_conf.yml:3]: is 4 spaces indentation intentionally or by accident?

File: files/galaxy/config/datatypes_conf.xml (hunk 1 above)
Comment (bgruening) [reply files/galaxy/config/datatypes_conf.xml:251]: In theory yes :)

File: files/galaxy/config/user_preferences_extra_conf.yml (hunk 2 above)
Comment (bgruening) [reply files/galaxy/config/user_preferences_extra_conf.yml:3]: intentionally, the upstream file is like that, not sure why.

**Review by mira-miracoli — APPROVED** [review]

File: files/galaxy/config/user_preferences_extra_conf.yml (hunk 2 above)
Comment (mira-miracoli) [reply files/galaxy/config/user_preferences_extra_conf.yml:3]: in this case we need to be careful in the future

**Pull request merged**